parser/prism/Translator.cc

```diff
@@ -748,7 +776,7 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node, bool preserveCon
 
             supportedCallType &= supportedBlock;
 
-            if (!supportedCallType) {
+            if (!directlyDesugar || !supportedCallType) {
```

Comment:
Forgot this in the first pass, but it was inconsequential.

Now some tests fail without it, idr the exact failure mode.

Don't worry about it.